[2024-05-02T00:28:05.256+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-02T00:28:05.570+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_btc_data.is_github_active scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-02T00:28:05.725+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_btc_data.is_github_active scheduled__2024-05-01T00:00:00+00:00 [queued]>
[2024-05-02T00:28:05.737+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-02T00:28:05.907+0000] {taskinstance.py:2327} INFO - Executing <Task(HttpSensor): is_github_active> on 2024-05-01 00:00:00+00:00
[2024-05-02T00:28:06.530+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1257) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-02T00:28:06.547+0000] {standard_task_runner.py:63} INFO - Started process 1295 to run task
[2024-05-02T00:28:06.667+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ingest_btc_data', 'is_github_active', 'scheduled__2024-05-01T00:00:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/ingest_btc_data.py', '--cfg-path', '/tmp/tmp4h83l0gy']
[2024-05-02T00:28:06.730+0000] {standard_task_runner.py:91} INFO - Job 58: Subtask is_github_active
[2024-05-02T00:28:06.924+0000] {task_command.py:426} INFO - Running <TaskInstance: ingest_btc_data.is_github_active scheduled__2024-05-01T00:00:00+00:00 [running]> on host 30e755686735
[2024-05-02T00:28:07.599+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='nononsaa.david@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingest_btc_data' AIRFLOW_CTX_TASK_ID='is_github_active' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-01T00:00:00+00:00'
[2024-05-02T00:28:07.606+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-02T00:28:07.663+0000] {baseoperator.py:399} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2024-05-02T00:28:07.677+0000] {http.py:137} INFO - Poking: kubit07/ingest_btc_data/kubit/btc_eur_data.csv
[2024-05-02T00:28:07.705+0000] {base.py:84} INFO - Using connection ID 'githubcon' for task execution.
[2024-05-02T00:28:08.005+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-05-02T00:28:08.016+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-02T00:28:08.053+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=ingest_btc_data, task_id=is_github_active, execution_date=20240501T000000, start_date=20240502T002805, end_date=20240502T002808
[2024-05-02T00:28:08.147+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-02T00:28:08.202+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-02T00:28:08.224+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
